js/dojoConfig.js

```diff
@@ -5,7 +5,7 @@ dojoConfig = {
   packages: [{
     name: "application",
     location: package_path + "/js/application",
-    main: "main"
+    main: "appInit"
```

Comment:
lets just rename to "init" because its already in the application folder. doesn't need "appInit". @MikeTschudi